server/pulp/server/webservices/views/serializers/link.py

```diff
@@ -0,0 +1,59 @@
+"""
+Generation of link objects for REST object serialization.
+
+link object:
+{
+    "_href": <uri path to resource or collection>
+}
+"""
+
+import copy
+
+from pulp.server.webservices import http
+
+
+_LINK_OBJ_SKEL = {
+    '_href': None,
+}
+
+
+def link_obj(href):
+    """
+    Create a link object for an arbitrary path.
+    @param href: uri path
```

Comment:
I'm not familiar with this `@param` style. Consider rewriting these to be more consistent with the rest of the codebase. This line would become:  `:param href: uri path`